### DISCUSSION TITLE
Cascading Folders

### DISCUSSION THREAD

**Discussion by beachnyc** [Open-Shell/Open-Shell-Menu#1139] [Q&A]
I apologize if this is an idiot question, but I cannot find a setting to make two folders on my start menu, "Downloads" and "Documents", to stop expanding. There are so many items in those folders that it takes forever to cascade and is just an annoyance. Is there a setting I am missing that would do this?

Thanks so much for any assistance.

**Comment by CTVCAM8** [comment]
Here are screenshots for Downloads without expanding menu, just the link.
I'll let you figure out Documents.

![Downloads no expand](https://user-images.githubusercontent.com/44127216/189766217-80a2e737-bc9d-4fab-bcdc-19b1180b5296.png)
